--- "src\\geomgraph\\DirectedEdgeStar.cpp.orig"	2017-10-30 09:41:21.112471400 +1000
+++ "src\\geomgraph\\DirectedEdgeStar.cpp"	2017-10-30 10:45:07.212511600 +1000
@@ -453,8 +453,8 @@
 {
 	std::string out="DirectedEdgeStar: " + getCoordinate().toString();
 
-	EdgeEndStar::iterator endIt=end();
-	for (EdgeEndStar::iterator it=begin(); it!=endIt; ++it)
+	EdgeEndStar::const_iterator endIt=end();
+	for (EdgeEndStar::const_iterator it=begin(); it!=endIt; ++it)
 	{
 		assert(*it);
         	assert(dynamic_cast<DirectedEdge*>(*it));
