--- tests\unit\capi\GEOSGeom_createCollection.orig.cpp	2016-11-14 14:10:07.895042000 +1000
+++ tests\unit\capi\GEOSGeom_createCollection.cpp	2016-11-14 14:14:19.742087000 +1000
@@ -106,7 +106,11 @@
         geoms.push_back(GEOSGeom_createEmptyPoint_r(handle_));
         // takes ownership of individual geometries
         geom_ = GEOSGeom_createCollection_r(handle_, GEOS_MULTIPOINT,
+#if defined(_MSC_VER) && (_MSC_VER == 1500)
+            &geoms[0], static_cast<int>(geoms.size()));
+#else
             geoms.data(), static_cast<int>(geoms.size()));
+#endif
         ensure_equals(GEOSGetNumGeometries_r(handle_, geom_), geoms.size());
     }
 
