--- geos-3.6.2.orig/tests/xmltester/XMLTester.cpp	2017-07-25 06:17:15.000000000 -0300
+++ geos-3.6.2/tests/xmltester/XMLTester.cpp	2017-10-26 18:56:27.377661886 -0300
@@ -338,6 +338,13 @@
     return old_value;
 }
 
+#if defined _MSC_VER && (_MSC_VER < 1900)
+double round(double number)
+{
+    return number < 0.0 ? ceil(number - 0.5) : floor(number + 0.5);
+}
+#endif
+
 /*private*/
 void
 XMLTester::printTest(bool success, const std::string& expected_result, const std::string& actual_result, const util::Profile &prof)
